Docker-FAQ.md

```diff
@@ -18,11 +18,11 @@ For some use-cases it is convenient to provide a modified `solr.in.sh` file to S
 For example to point Solr to a ZooKeeper host:
 
 ```
-docker create --name my-solr -P solr
-docker cp my-solr:/opt/solr/bin/solr.in.sh .
+docker create --name my_solr -P solr
+docker cp my_solr:/opt/solr/bin/solr.in.sh .
 sed -i -e 's/#ZK_HOST=.*/ZK_HOST=cylon.lan:2181/' solr.in.sh
```

Comment:
Do we really want to make it look this hard vs. simply setting `-e ZK_HOST=CYLON.LAN:2181` ?